project/AkkaBuild.scala

```diff
@@ -1332,12 +1329,20 @@ object AkkaBuild extends Build {
       ProblemFilters.exclude[MissingTypesProblem]("akka.remote.ReliableDeliverySupervisor$GotUid$"),
       ProblemFilters.exclude[MissingMethodProblem]("akka.remote.ReliableDeliverySupervisor#GotUid.apply")
     )
+
+    val akkaStream = Seq(
```

Comment:
These are incompatibilities between `2.0` and current `stream-http-2.0` (currently same as `2.0.1`) therefore these incompatibilities need to be filtered out.